libs/MotionKit/tests/MotionKit_test.cpp

```diff
@@ -59,13 +58,10 @@ TEST_F(MotionKitTest, initialization)
 
 TEST_F(MotionKitTest, registerMockCallbackAndRotate)
 {
-	auto mock_function_imu = MockFunction<void(void)> {};
-	auto loop_imu		   = [&] { mock_function_imu.Call(); };
-
 	auto mock_function_motion = MockFunction<void(void)> {};
 	auto loop_motion		  = [&] { mock_function_motion.Call(); };
 
-	EXPECT_CALL(mock_function_imu, Call()).Times(1);
+	EXPECT_CALL(lsm6dsox, setPowerMode(interface::LSM6DSOX::PowerMode::Normal)).Times(1);
```

Comment:
could you comment out all of MotionKit tests? 

because of all the changes it's not in a state where the tests can be meaningful.

and if it means adding lines like this one, it means that something wrong. Motionkit should only rely on IMUKit.

Reply:
at least add `// TODO(@leka/dev-embedded): temporary fix, remove when updating fusion algorithm`